testing/test_mark.py

```diff
@@ -1192,19 +929,12 @@ class TestBarClass(BaseTests):
     deselected_tests = dlist[0].items
     assert len(deselected_tests) == 1
 
+    # todo: fixed
     # keywords smear - expected behaviour
-    reprec_keywords = testdir.inline_run("-k", "FOO")
-    passed_k, skipped_k, failed_k = reprec_keywords.countoutcomes()
-    assert passed_k == 2
-    assert skipped_k == failed_k == 0
-
-
-def test_addmarker_getmarker():
-    node = Node("Test", config=mock.Mock(), session=mock.Mock(), nodeid="Test")
-    node.add_marker(pytest.mark.a(1))
-    node.add_marker("b")
-    node.get_marker("a").combined
-    node.get_marker("b").combined
+    # reprec_keywords = testdir.inline_run("-k", "FOO")
```

Comment:
Remove commented code?

Reply:
that oe i left commented for now since i want to take a look at the keyword interaction - i'd like to avoid a breaking change